samples/sample_hevc_fei/CMakeLists.txt

```diff
@@ -17,7 +17,6 @@ include_directories (
 list( APPEND LIBS_VARIANT sample_common )
 
 set(DEPENDENCIES ${skipping} itt libmfx dl pthread)
-set( defs "-Werror" )
```

Comment:
Originally samples had separate build infrastructure and didn't use -Werror and didn't support WARNING_FLAGS. Now WARNING_FLAGS can be used, so better to make your change in order to your commit message, i.e. set WARNING_FLAGS in CMakeLists.txt for sample_hevc_fei.

Reply:
Ok.  I was under the impressions that WARNING_FLAGS was handled at a higher/top level of the project configuration.  I will double check.